examples/basic-example/requirements.txt

```diff
@@ -0,0 +1 @@
+mirascope==0.1.0
```

Comment:
this also requires openai

Reply:
does installing mirascope not install openai?